src/code.rs

```diff
@@ -80,16 +80,28 @@ pub struct StructField {
     pub base_type: String,
     /// Indicate that this field is optional
     pub is_optional: bool,
+    /// Indicate that this field is a vec
+    pub is_vec: bool,
```

Comment:
what do you think about refactoring `StructField` to have a optional lower type which can have its own options (and so be optionally "recursive"), at least in the future?

for example if postgres should ever change it, or for other backends that handle it differently